Automattic/jetpack
# My Jetpack: do not display Manage banner for agency customers

Fixes #40226

## Proposed changes:

Do not display the Jetpack Manage banner to accounts that have been enrolled into our agency program.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?
* No

## Testing instructions:

* Start with a site connected to WordPress.com.
* Go to Jetpack > My Jetpack
* Scroll down to the bottom of the page.
* Check if the Jetpack Manage banner is there, based on your WordPress.com account properties.
